addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -47,6 +47,18 @@ options.Class.include({
 });
 
 options.registry.background.include({
+    background: async function (previewMode, widgetValue, params) {
```

Comment:
In fact, in this second commit, maybe you could already adapt the cover properties option (used by blog and event) ? Right now, there is two custom buttons "Change cover" and "Remove cover", it should simply be one "Cover" we-imagepicker. Your PR would remove even more lines \o/